build.gradle

```diff
@@ -14,11 +14,25 @@ description = "tapestry-offline"
 
 repositories {
      mavenCentral()
+     maven {
+     	credentials {
+            username mavenUser
+            password mavenPassword
+        }
+        url "https://nexus.pubfactory.com/nexus/content/repositories/releases"
+     }
+     maven {
+     	credentials {
+            username mavenUser
+            password mavenPassword
+        }
+        url "https://nexus.pubfactory.com/nexus/content/repositories/snapshots"
+     }
```

Comment:
do we actually need the gradle configuration at all if this is a maven project?  I'm just hoping that for people to work on this project they don't need to edit this file with their maven username/password because that's just asking for people to commit their actual passwords to git.

Reply:
We don't actually need it no.  I kept it around because I thought I was going to do a PR to the project itself, but then realized I needed to downgrade tapestry.

mavenUser and mavenPassword are properties you can set in your ~/.gradle/gradle.properties file so you don't actually change this file.